saleor/graphql/attribute/utils.py

```diff
@@ -605,10 +605,13 @@ def _pre_save_numeric_values(
         else:
             return tuple()
 
+        slug = slugify(unidecode(f"{value}_{attribute.id}"))
```

Comment:
It's not the right solution. The task what to investigate `why multiple rows are created` and not to change the number attribute slug pattern. 
We have two different types of attributes:
- `TYPES_WITH_CHOICES`
- `TYPES_WITH_UNIQUE_VALUES`
The numeric attribute is the type with unique values.

With your solution, two different instances of le's say `Product` may start sharing the `AttributeValue`. When one of these products is deleted, automatically `AttributeValue` is deleted, which means that we will end up with an empty value for attribute for the second `Product`.

Reply:
Hmm, are you sure that we delete shared attribute values? I wrote a new test, which shows, that with the solution:

1. we don't create duplicated attribute values anymore
2. after deleting one of the instances, which share the same attribute value, the value is still associated with remaining instances

Do I miss something? Can you please check it again?

Reply:
Here is the method that is responsible for that:
https://github.com/saleor/saleor/blob/main/saleor/graphql/product/mutations/product/product_delete.py#L85